### ISSUE TITLE
Symfony 2.7 - template assets helper error

### ISSUE THREAD

**Issue by mpclarkson** [liip/LiipMonitorBundle#103]
Hi there. There appears to be a bug on dev-master and v2.0.8 with Symfony 2.7 Beta at line 17 in `liip/monitor-bundle/Liip/MonitorBundle/Helper/PathHelper.php at line 17 `

```
Catchable Fatal Error: Argument 1 passed to Symfony\Bundle\FrameworkBundle\Templating\Helper\AssetsHelper::__construct() must be an instance of Symfony\Component\Asset\Packages, string given...
```

**Comment by xabbuh** [comment]
This was fixed in symfony/symfony#14419 and will be included in the next beta/the stable release.

**Comment by lsmith77** [comment]
thx ..
